### PR TITLE
Adds on examine hints for constructing turrets similar to walls or mechs

### PR DESCRIPTION
**Pull request by itseasytosee**
## About The Pull Request
Just adds some on examine text depending on the turrets stage of construction to guide you on what do do while building a turret

## Why It's Good For The Game
People no longer have to open the wiki to make turrets

## Changelog
:cl: itseasytosee
qol: Adds some on examine instructions for constructing a turret
/:cl:

### CODE REVIEW

File: code/game/machinery/porta_turret/portable_turret_construct.dm [hunk 1]
```diff
@@ -19,6 +19,28 @@
...
+		if(PTURRET_UNSECURED)
+			. += span_notice("The external bolts are <b>unwrenched</b>, and the frame could be <i>pried</i> apart.")
+		if(PTURRET_BOLTED)
+			. += span_notice("The external bolts are <i>wrenched</i> in place, the frame requires <b>metal</b> for its internal armor.")
+		if(PTURRET_START_INTERNAL_ARMOUR)
+			. += span_notice("The turret's armor needs to be <b>bolted</b> in place, the armor looked like it could be <i>welded</i> out.")
+		if(PTURRET_INTERNAL_ARMOUR_ON)
+			. += span_notice("The turret requires an <b>energy based gun</b> to fuction, the armor is secured by <i>bolts</i>.")
+		if(PTURRET_GUN_EQUIPPED)
+			. += span_notice("The turret requires an <b>proximity sensor</b> to fuction.")
```
[the remark this answers is not in the document]
Comment (MrDoomBringer) [reply code/game/machinery/porta_turret/portable_turret_construct.dm:34]: Also you misspelled function

File: code/game/machinery/porta_turret/portable_turret_construct.dm [hunk 2]
```diff
@@ -19,6 +19,28 @@
...
+	. = ..()
+	switch(build_step)
+		if(PTURRET_UNSECURED)
+			. += span_notice("The external bolts are <b>unwrenched</b>, and the frame could be <i>pried</i> apart.")
+		if(PTURRET_BOLTED)
+			. += span_notice("The external bolts are <i>wrenched</i> in place, the frame requires <b>metal</b> for its internal armor.")
+		if(PTURRET_START_INTERNAL_ARMOUR)
+			. += span_notice("The turret's armor needs to be <b>bolted</b> in place, the armor looked like it could be <i>welded</i> out.")
+		if(PTURRET_INTERNAL_ARMOUR_ON)
+			. += span_notice("The turret requires an <b>energy based gun</b> to fuction, the armor is secured by <i>bolts</i>.")
```
Comment (MrDoomBringer) [inline code/game/machinery/porta_turret/portable_turret_construct.dm:32]:
```suggestion
			. += span_notice("The turret requires an <b>energy based gun</b> to function, the armor is secured by <i>bolts</i>.")
```

File: code/game/machinery/porta_turret/portable_turret_construct.dm [hunk 3]
```diff
@@ -19,6 +19,28 @@
...
+		if(PTURRET_GUN_EQUIPPED)
+			. += span_notice("The turret requires an <b>proximity sensor</b> to fuction.")
+		if(PTURRET_SENSORS_ON)
+			. += span_notice("The turret's access hatch is <b>unscrewed</b>.")
```
Comment (MrDoomBringer) [inline code/game/machinery/porta_turret/portable_turret_construct.dm:36]: Ditto 
```suggestion
			. += span_notice("The turret's access hatch is <b>unscrewed</b>. The proximity sensor could <i>be removed</i>.")
```

**Review by MrDoomBringer — CHANGES_REQUESTED** [review]

**Comment by itseasytosee** [comment]
I did not know you could take the prox sensor out, go figure.

**Comment by Rohesie** [comment]
Good PR!

**Pull request merged**